docs/sources/developers/plugins/sign-a-plugin.md

```diff
@@ -12,9 +12,7 @@ Before you can sign your plugin, you need to decide whether you want to sign it
 
 If you want to make your plugin publicly available outside of your organization, you need to sign your plugin under a _community_ or _commercial_ [signature level](#plugin-signature-levels). Public plugins are available from [grafana.com/plugins](https://grafana.com/plugins) and can be installed by anyone.
 
-For more information on how to install a public plugin, refer to [Install Grafana plugins]({{< relref "../../administration/plugin-management#install-a-plugin" >}}).
```

Comment:
Are you intentionally removing this information from the file?

Reply:
Yes, this sentence is not germaine to the procedure.